### DISCUSSION TITLE
Console access error (404 page not found)

### DISCUSSION THREAD

**Discussion by L-Andy** [appwrite/appwrite#3866] [Q&A]
Hello appwriters,
As we were all excited about the v1 of apppwrite, but yesterday i got the fresh installation and trying to access the console it is throwing the 404 page not found error.
I have done the docke container prune.
Cleaned all the docker images and containers and reinstalled everything but all in vain.
Any help will be of great use.

**Comment by L-Andy** [comment]
After diging through traefik logs i got found this error
![8](https://user-images.githubusercontent.com/54024696/190851435-32fefe12-59fb-4da5-a7cf-55b5af8fcd43.png)

**Reply by stnguyen90** [reply]
How did you clean up your old instance of Appwrite?
